pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -160,51 +188,132 @@ const CryptoPolicyDialog = ({
-                       <Button id="crypto-policy-save-reboot" variant='primary' onClick={() => setPolicy(true)}>
+                       {inProgress &&
+                           <Flex spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }} className="fips-mode-progress">
+                               <Spinner isSVG size="md" />
+                               <p>{_("Applying FIPS policy... This may take a few minutes.")}</p>
+                           </Flex>}
+                       <Button id="crypto-policy-save-reboot" variant='primary' onClick={() => setPolicy(true)}
+                               isDisabled={inProgress} isLoading={rebootRequired && inProgress}
+                       >
                            {_("Save and reboot")}
                        </Button>
-                       <Button id="crypto-policy-save-reboot-later" variant='secondary' onClick={() => setPolicy(false)}>
+                       <Button id="crypto-policy-save-reboot-later" variant='secondary' onClick={() => setPolicy(false)}
+                               isDisabled={inProgress} isLoading={!rebootRequired && inProgress}
+                       >
                            {_("Save only")}
                        </Button>
-                       <Button variant='link' onClick={close}>
+                       <Button variant='link' onClick={close} isDisabled={inProgress}>
                            {_("Cancel")}
                        </Button>
                    </>
                }
         >
             {currentCryptoPolicy && <ProfilesMenuDialogBody active_profile={currentCryptoPolicy}
                                                      change_selected={setSelected}
+                                                     isDisabled={inProgress}
                                                      profiles={policies} />}
         </Modal>
     );
 };
 
 export const CryptoPolicyStatus = () => {
     const [requiresReboot, setRequiresReboot] = useState(false);
+    const [currentCryptoPolicy, setCurrentCryptoPolicy] = useState(null);
+    const [fipsEnabled, setFipsEnabled] = useState(null);
     const [showShutdownModal, setShowShutDownModal] = useState(false);
+    const [showEnableFIPSModal, setShowEnableFIPSModal] = useState(false);
 
     useEffect(() => {
         if (!requiresReboot) {
             // For when we change the setting
             cockpit.file(applyNeedRebootStamp).watch(content => setRequiresReboot(content !== null));
         }
-    }, [requiresReboot]);
 
-    if (!requiresReboot) {
-        return null;
+        // Avoid cockpit.spawn every re-render from parent
+        if (currentCryptoPolicy === null) {
+            cockpit.spawn(["update-crypto-policies", "--show"], { err: "message" })
+                    .then(output => setCurrentCryptoPolicy(output.trim()))
+                    .catch(err => {
+                        console.debug("no crypto policies support: ", err.toString());
```

Comment:
Let's please take this out. I am now seeing this message in each test log (e.g. open any test in [a recent log](https://logs.cockpit-project.org/logs/pull-17176-20220324-101200-bb8921de-debian-stable/log.html)). There must be another similar message around in the code, as this one is new, and this PR does not just move it -- this is in addition.

Reply:
It's still in the current version, but opening a new thread.